components/Header.js

```diff
@@ -0,0 +1,15 @@
+import styled from "styled-components";
+
+export default function Header() {
+  return <StyledHeader>Voc-Trainer</StyledHeader>;
```

Comment:
Could it be good to wrap the headline in an `<h1>` tag for semantic reasons?

Reply:
This is an interesting topic that I read some more about.

[MDN writes](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements?retiredLocale=de#:~:text=a%20page%20should%20generally%20have%20a%20single%20%3Ch1%3E%20element%20that%20describes%20the%20content%20of%20the%20page%20(similar%20to%20the%20document's%20%3Ctitle%3E%20element).): "A page should generally have a single `<h1>` element that describes the content of the page (similar to the document's `<title>` element)." The app name doesn't describe the content of the page, though, so I'd say the `<header>` tag is sufficient after all. To comply with the convention of having an `<h1>` on every page, you would probably have to add a heading ("Your vocabulary today" or similar) in the main. But that will be another user story. :D

By the way: In my opinion, styling should not play a role in the decision of heading tags, because if I understand it correctly, the heading tags are primarily important for screen readers and SEO.